tbx/static_src/sass/components/navigation/_sub-nav-desktop-mini.scss

```diff
@@ -43,7 +43,8 @@ Here we refer to 'level 1 children' as 'level 2' and 'level 2 children' as 'leve
     &__link {
         @include link-styles(
             $color: var(--color--heading),
-            $interaction-color: var(--color--heading)
+            $interaction-color: var(--color--heading),
+            $underline-color: var(--color--heading)
```

Comment:
I think the navs also needs to change to the underline colour, if we are to follow ben's request (check contrast though)

Reply:
The main nav should be updated. The new colour he suggested using doesn't pass colour contrast on the child links in the dropdown's that use a different background colour.